static/src/javascripts/bootstraps/enhanced/article.js

```diff
@@ -19,6 +19,7 @@ import { initTrails } from 'bootstraps/enhanced/trail';
 import ophan from 'ophan/ng';
 import { SnippetFeedback } from 'journalism/snippet-feedback';
```

Comment:
is `journalism` the right name here? will that make most sense if/when the journalism okr is shut down?

Reply:
This is one of the modules that will go away when we're confident the library works fine. Ok to leave it like that until then?

Reply:
if there's stuff that is only temporary, it would be really good to mark as such (they would be the first 'temporary' files to be left in place – speaking from personal experience here 😉)

maybe `'journalism-okr-temporary/snippet-feedback'` or some such?

Reply:
You're right, there was definitely things we can already get rid of too: https://github.com/guardian/frontend/pull/18264